packages/help-center/src/components/help-center-header.tsx

```diff
@@ -57,13 +57,19 @@ const SupportModeTitle = () => {
 
 const Content = ( { onMinimize }: { onMinimize?: () => void } ) => {
 	const { __ } = useI18n();
+	const { pathname } = useLocation();
+
+	const getHeaderText = () => {
+		if ( pathname === '/odie' || pathname === '/contact-form' ) {
+			return __( 'Virtual Expert', __i18n_text_domain__ );
+		}
+		return __( 'Help Center', __i18n_text_domain__ );
+	};
```

Comment:
I think we may want to align this with the minimized header as well. Currently it shows "Wapuu" when minimizing.